vscode/snippets/snippets.json

```diff
@@ -0,0 +1,72 @@
+{
+    "lambda": {
+        "prefix": "lambda",
+        "description": "home",
```

Comment:
This description (`"home"`) doesn't make sense - copy-paste error?

I'm a bit confused by how the the object key (`"lambda"`) and the description (`"home"`) differ - they seem to be both human-readable description strings. AFAIK it works like this in VS Code:

- If the description is defined, it is used as the description. The object key doesn't seem to be used anywhere
- If the description is not defined, the object key is used as the description.

So it seems to me that just using the object key as the description is sufficient. I'll remove the description fields.